cerebrium/advanced-functionality/using-webhook-endpoints.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Using Webhook Endpoints for Async Results"
+---
+
+Webhook endpoints can be used to obtain asynchronous results from your requests. This is useful when your model takes longer than the max timeout (180 seconds) or when you don't want to wait for the model to finish processing the request and prefer receiving the response once the processing is completed.
+
+To use this feature, simply include the `webhook_endpoint` field in your request body, assigning it to your desired URL. As soon as this is done, your requests will return immediately with the `run_id`. After processing, a POST request containing the results and `run_id` will be made to the provided webhook_endpoint URL.
+
+<Note>
+    This feature is available on all model types: custom & pre-built. If you have any questions on how to use it please
+    reach out to us on Slack or Discord
+</Note>
+
+Here is an example on how to add a webhook endpoint to your request body:
+
+```json
+{
+"data": [...],
+"webhook_endpoint": "https://your-webhook-url.com"
+}
+```
```

Comment:
Is there a way to clarify this a bit for someone who is a complete beginner on our system?
In the rest of the docs, they get:
      --data-raw '{
        "prompt": "some prompt"
    }'
    
  So does it make sense to change this to: (I took this from whisper's docs)
        --data-raw '{
        "data: "<your data>"
        "webhook_endpoint": "<WEBHOOK_ENDPOINT>",
        }'

Reply:
ok, updated to include a full curl request (minus api key)